NuGet.config

```diff
@@ -2,8 +2,9 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="NuGet2" value="https://api.nuget.org/v3/index.json" />
+    <!-- <add key="Core" value="http://dotnet.myget.org/F/dotnet-core" /> -->
```

Comment:
Can you do the same thing we do with nuget config files: https://github.com/aspnet/IISIntegration/blob/dev/NuGet.config
and https://github.com/aspnet/IISIntegration/blob/dev/build/sources.props

Reply:
Including in next revision